source/manual/review-apps.html.md

```diff
@@ -0,0 +1,166 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-05-20
+title: Review apps for Pull Requests
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+# Review apps for Pull Requests
+
+Review apps allow us to automatically deploy application changes into a brand
+new Heroku app so that each Pull Request reviewer has a chance to see how it
+looks with live data.
+
+There are currently 2 examples of GOV.UK apps that have review apps enabled:
+- [collections](https://github.com/alphagov/collections)
+- [government-frontend](https://github.com/alphagov/government-frontend)
+
+Follow the steps below in order to enable review apps for your GOV.UK
+application.
+
+## Create a Heroku account
+
+If you don't have a Heroku account yet, please create one using your work email
+address.
+
+You can do so on this page: https://signup.heroku.com
+
+## Install the Heroku Toolbelt
+
+Install the Toolbelt (instructions on
+https://devcenter.heroku.com/articles/heroku-cli) and login on your shell:
+
+```
+$ heroku login
+```
+
+Use the credentials of the account created above when prompted.
+
+## Create a new Heroku app
+
+Login to Heroku and create a new application in the European availability zone.
+Give the app a meaningful name to closely match the application name on GitHub.
+
+Then, in your shell, go into the application directory and add the Heroku
+remote:
+
+```
+$ cd ~/govuk/govuk-app
+$ heroku git:remote -a govuk-app
+```
+
+This will allow you to deploy the app to Heroku.
+
+## Enable your app to work on Heroku
+
+There are 2 things needed to be configured before you can deploy your app to
+Heroku:
+
+- Add a [Procfile](https://devcenter.heroku.com/articles/procfile) that tells
+heroku how to run your application;
+- Add a file called `app.json` with the expected app configuration.
+
+An example Procfile for a standard Rails application will look something like
+this:
+
+```
+web: GOVUK_ASSET_ROOT=https://${HEROKU_APP_NAME}.herokuapp.com bin/rails server -p $PORT -e $RAILS_ENV
```

Comment:
I wonder if this is actually needed, since smart answer apps work without it:

https://github.com/alphagov/smart-answers/blob/master/startup_heroku.sh

Won't the default `config.asset_host` work?

http://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html